pandas/tests/dtypes/test_inference.py

```diff
@@ -13,7 +13,8 @@
 import numpy as np
 import pytz
 import pytest
-
+from numbers import Number
```

Comment:
can you put the builtin imports at the top

Reply:
ok, sure!